main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
```

Comment:
Please swap this with the one from your project as a good convention the imports should go like
core -> packages from projects -> third-party packages such as mux.

Reply:
Got it! Updated